Makefile.COMMON

```diff
@@ -44,12 +44,17 @@ endif
 GO_VERSION ?= 1.4.1
 GOURL      ?= https://golang.org/dl
 GOPKG      ?= go$(GO_VERSION).$(GOOS)-$(GOARCH)$(RELEASE_SUFFIX).tar.gz
-GOROOT     := $(CURDIR)/.build/go
 GOPATH     := $(CURDIR)/.build/gopath
 GOCC       ?= $(GOROOT)/bin/go
 GO         ?= GOROOT=$(GOROOT) GOPATH=$(GOPATH) $(GOCC)
 GOFMT      ?= $(GOROOT)/bin/gofmt
 
+ifeq ($(shell go version | sed 's/.*go\([0-9.]*\).*/\1/'), $(GO_VERSION))
```

Comment:
How about adding a `2>/dev/null` to the `go version`. Then someone won't see an error outputted if they don't have any Go in their PATH at all.